### PR TITLE
Update set-up-new-project.md

### PR DESCRIPTION
**Pull request by firtina-improbable**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Was doing a quick test to see if this would work, found some problems to solve.
#### Tests
Testing with a blankish project
#### Documentation
You're looking at it
#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: docs/content/set-up-new-project.md [hunk 1]
```diff
@@ -8,6 +8,16 @@ Follow the [Setup and installation guide](../setup-and-installing.md) to make su
...
 To use the SpatialOS GDK for Unity in a new project, you need to set up your project manifest, and then set up world initialization. Your new project must have the [same structure as a SpatialOS project](https://docs.improbable.io/reference/latest/shared/reference/project-structure).
 
+The **spatialos.json** file for your project needs to have the same `sdk_version` and `dependencies` as the GDK's [spatialos.json](../../spatialos.json).
+
+For a basic set up of two workers, one `UnityGameLogic` and one `UnityClient`, we recommend you to reuse these files (within `workers/unity`):
+- [spatialos.UnityGameLogic.worker.json](../../workers/unity/spatialos.UnityGameLogic.worker.json)
+- [spatialos.UnityClient.worker.json](../../workers/unity/spatialos.UnityClient.worker.json)
+
+### Set up base assets and directories
+- Copy [this file](../../workers/unity/Assets/Config/GdkToolsConfiguration.json) into `workers/unity/Assets/Config/GdkToolsConfiguration.json`.
```
Comment (jessicafalk) [inline docs/content/set-up-new-project.md]: interesting, though this would be automatically generated? need to check with Jamie for that

Comment (firtina-improbable) [reply docs/content/set-up-new-project.md]: Adding @jamiebrynes7 ☝️

Comment (firtina-improbable) [reply docs/content/set-up-new-project.md]: I had an error when I didn't have it, but can check what happens if I follow everything else but this

File: docs/content/set-up-new-project.md [hunk 2]
```diff
@@ -75,4 +85,4 @@ namespace YourProject
 This sets up the injection hooks needed to run Unity's hybrid ECS and ensures that all worlds are properly cleaned up. You need to set the `initialized` field to `true` to ensure it is only run once, otherwise `Init` is run whenever a scene gets loaded due to the `RuntimeInitializeOnLoadMethod` attribute.
 
 ----
-**Give us feedback:** We want your feedback on the SpatialOS GDK for Unity and its documentation  - see [How to give us feedback](../../README.md#give-us-feedback).
\ No newline at end of file
+**Give us feedback:** We want your feedback on the SpatialOS GDK for Unity and its documentation  - see [How to give us feedback](../../README.md#give-us-feedback).
```
Comment (firtina-improbable) [inline docs/content/set-up-new-project.md]: probably a newline change at end of file, I had used GitHub's in-place editing

File: docs/content/set-up-new-project.md (hunk 1 above)
Comment (jamiebrynes7) [reply docs/content/set-up-new-project.md]: This should be auto generated. If its not that's a bug report

Comment (firtina-improbable) [reply docs/content/set-up-new-project.md]: Just deleted it and can confirm it was indeed re-generated

File: docs/content/set-up-new-project.md [hunk 3]
```diff
@@ -8,6 +8,15 @@ Follow the [Setup and installation guide](../setup-and-installing.md) to make su
 
 To use the SpatialOS GDK for Unity in a new project, you need to set up your project manifest, and then set up world initialization. Your new project must have the [same structure as a SpatialOS project](https://docs.improbable.io/reference/latest/shared/reference/project-structure).
 
+The **spatialos.json** file for your project needs to have the same `sdk_version` and `dependencies` as the GDK's [spatialos.json](../../spatialos.json).
+
+For a basic set up of two workers, one `UnityGameLogic` and one `UnityClient`, we recommend you to reuse these files (within `workers/unity`):
+- [spatialos.UnityGameLogic.worker.json](../../workers/unity/spatialos.UnityGameLogic.worker.json)
+- [spatialos.UnityClient.worker.json](../../workers/unity/spatialos.UnityClient.worker.json)
```
Comment (jamiebrynes7) [inline docs/content/set-up-new-project.md]: Are we expecting users to have the worker folder named the same thing? Afaik this is generally disliked

Comment (firtina-improbable) [reply docs/content/set-up-new-project.md]: discussed offline, `workers/unity` is mentioned in other parts of docs, so may need to be done with another PR

**Pull request merged**